### PR TITLE
DCOS-12290: (UCR) - Add bridge network support

### PR DESCRIPTION
**Pull request by Poltergeist**
**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [x] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

Add Bridge mode network support for ucr
single container apps.

Closes DCOS-12290

### CODE REVIEW

File: plugins/services/src/js/reducers/serviceForm/__tests__/Container-test.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ describe('Container', function () {
             image: 'foo',
             forcePullImage: null,
             privileged: null,
-            network: undefined,
```
Comment (mlunoe) [inline plugins/services/src/js/reducers/serviceForm/__tests__/Container-test.js]: Could we add tests for UCR and BRIDGE to test that the output is as expected?

**Comment by nLight** [comment]
What am I doing wrong?

screen shot 2017-04-20 at 12 55 32

**Comment by Poltergeist** [comment]
This feature is yet not implemented into Marathon, yet. Next version should contain this feature after that we can also move this feature to master.

File: plugins/services/src/js/reducers/serviceForm/__tests__/Container-test.js (hunk 1 above)
Comment (Poltergeist) [inline plugins/services/src/js/reducers/serviceForm/__tests__/Container-test.js]: Added and actually fixed the tests. Unfortunately a lot of the container tests were not executed.

**Comment by nLight** [comment]
@mlunoe could you please give it another shot?

**Comment by mlunoe** [comment]
Looks good and works well!

**Pull request merged**